hack/azblob_test/test.sh

```diff
@@ -9,12 +9,14 @@ BLOB_PORT=10000
 azurite --silent --location /tmp/azurite --debug /tmp/azurite/azurite.debug --blobPort ${BLOB_PORT} &
 timeout 15 bash -c "until echo > /dev/tcp/localhost/${BLOB_PORT}; do sleep 0.5; done"
 
-buildkitd -debugaddr 0.0.0.0:8060 &
-while true; do
-  curl -s -f http://127.0.0.1:8060/debug/pprof/ >/dev/null && break
+buildkitd &
+while [[ ! -S "/run/buildkit/buildkitd.sock" ]]; do
```

Comment:
Is there any chance we could add a timeout / max-retries counter, similar to how it's done in https://github.com/moby/buildkit/blob/master/examples/buildctl-daemonless/buildctl-daemonless.sh#L41?

Ideally we shouldn't keep the job running if buildkit hasn't started after some "reasonable" amount of time :eyes:

Reply:
excellent suggestion! will use the same